test/PowerShellEditorServices.Test.E2E/LanguageServerProtocolMessageTests.cs

```diff
@@ -35,12 +35,12 @@ public class LanguageServerProtocolMessageTests : IClassFixture<LSPTestsFixture>
 
         public LanguageServerProtocolMessageTests(ITestOutputHelper output, LSPTestsFixture data)
         {
-            Diagnostics = new List<Diagnostic>();
             LanguageClient = data.LanguageClient;
             Diagnostics = data.Diagnostics;
-            PwshExe = TestsFixture.PwshExe;
             Diagnostics.Clear();
 
+            PwshExe = TestsFixture.PwshExe;
```

Comment:
what's the reason for this change?

Reply:
I needed to edit the tests and moved this down so it wasn't with the list instantiation code